fintech-examples/fintech-ui/src/app/interceptors/auth.interceptor.ts

```diff
@@ -17,9 +18,11 @@ export class AuthInterceptor implements HttpInterceptor {
   }
 
   private handleRequest(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
-    const headers = new HttpHeaders()
-      // TODO: to be defined
-      .set('X-Request-ID', '99391c7e-ad88-49ec-a2ad-99ddcb1f7721')
+    for (const key of request.headers.keys()) {
+      console.log('orig request has header key', key, ' with value: ', request.headers.get(key));
+    }
```

Comment:
let's probably not commit it. We can inspect the headers anyways when we send a request